src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java

```diff
@@ -33,8 +33,9 @@ public static void main(String[] args) {
      * @param milliseconds время (в миллисекундах) между срабатыванием датчика света и звука
      */
     static float getDistance(float milliseconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+        float newTime = milliseconds / 1000;
+        float newSpeed = (SOUND_SPEED * 5) / 18;
+        float Distance = newSpeed * newTime;
```

Comment:
"Violation: 'nameVariable' was not used for variable naming
Solution: Rename variable"